target/linux/ramips/rt305x/base-files/etc/board.d/02_network

```diff
@@ -164,6 +164,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:wan" "6@eth0"
 		;;
+	zte,mf283plus)
+		ucidef_add_switch "switch0" \
+			"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "6@eth0"
```

Comment:
Is the right side of the numbering really necessary?

I'd expect this to give the same result in LuCI:

`"0:lan" "1:lan" "2:lan" "3:lan" "6@eth0"`

Reply:
Indeed it does, fixed.